numpy/core/src/umath/ufunc_object.c

```diff
@@ -5693,18 +5693,13 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
      * Create dtypes array for either one or two input operands.
      * The output operand is set to the first input operand
      */
-    dtypes[0] = PyArray_DESCR(op1_array);
     operands[0] = op1_array;
     if (op2_array != NULL) {
-        dtypes[1] = PyArray_DESCR(op2_array);
-        dtypes[2] = dtypes[0];
```

Comment:
Why are these no longer assigned to?

Reply:
Because they should be filled in by the TypeResolver in any case, initializing it doesn't really do anything.